src/main/java/gov/usgs/owi/nldi/services/ConfigurationService.java

```diff
@@ -42,4 +41,14 @@ public String getRootUrl() {
 	public String getAppVersion() {
 		return appVersion;
 	}
+
+	public String getSwaggerApiDocsUrl() {
+		return getDisplayPath() + "/v3/api-docs/swagger-config";
```

Comment:
If you are hard coding this here, the configurable value in the application.yml is not needed.